### PR TITLE
Enhancements1

### PR DESCRIPTION
**Pull request by razorsedge**
- Ship crudini to support the CSD.
- Ship mkuser.py in the parcel and not the CSD.
- Setup /etc/airflow/conf.dist.
- Lint and validate JSON configs.
- When building, do not download files if they already exist.
- Pin Pandas and Flask to fix build/run issues.
- Create wrapper scripts to set the correct environment for various programs (airflow, crudini, airflow-mkuser).

Requires teamclairvoyant/apache-airflow-cloudera-csd#5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 This repository allows you to install [Apache Airflow](https://airflow.apache.org/) as a parcel deployable by [Cloudera Manager](https://www.cloudera.com/products/product-components/cloudera-manager.html).
 
+## Requirements
+- A supported operating system.
+- MySQL or PostgreSQL database in which to store Airflow metadata.
+
+### Currently Supported Versions of Airflow
+- Airflow 1.10
```
Comment (rssanders3) [inline README.md:10]: versions available:
1.10
1.10.1
1.10.3

**Review by rssanders3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (razorsedge) [reply README.md:10]: Do we want to update the README every time we build? I left out patch-level versions just for that purpose.

**Comment by thesuperzapper** [comment]
@razorsedge @rssanders3 please also consider using my PR as a base, as it addresses many issues with the current process, and removes the need to spam the local docker repo with partially built images. 

You can see the PR here: #3

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — COMMENTED** [review]
"airflow_env.sh" is confusing with "airflow-env.sh". Should we consider renaming it to something less confusing?

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — APPROVED** [review]

**Review by Naveen481 — COMMENTED** [review]
We are currently invoking airflow.sh from /usr/bin/airflow. Within airflow.sh we will be setting the required environment variables incase if the user wants to manually use airflow command.

**Review by Naveen481 — APPROVED** [review]

**Comment by razorsedge** [comment]
@Naveen481 Those names follow the standards in CDH.  The CDH parcel ships `meta/cdh_env.sh` and the Hadoop CLIs read `/etc/hadoop/conf/hadoop-env.sh`  `meta/airflow_env.sh` is only referenced internally to CM.  `/etc/airflow/conf/airflow-env.sh` is referenced by Airflow CLI tools.

File: README.md (hunk 1 above)
Comment (rssanders3) [reply README.md:10]: Ok that should be fine

**Review by rssanders3 — APPROVED** [review]

**Pull request merged**